src/contexts/search-context.ts

```diff
@@ -37,6 +38,7 @@ export function createSearchContext(
     placeholderSearch: options.placeholderSearch !== false, // true by default
     keepZeroFacets: !!options.keepZeroFacets, // false by default
     finitePagination: !!options.finitePagination, // false by default
+    matchingStrategy: options.matchingStrategy || MatchingStrategies.LAST,
```

Comment:
The default behavior can be handled by Meilisearch, why do we need to set it here?

Reply:
Mostly safeguarding, but indeed not necessary. Would you like me to remove it?

Reply:
Just thinking, if this is good or not, if in the future meilisearch decides to change the default value, we will have differences between SDKs...